amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerSubscriber.java

```diff
@@ -73,6 +73,7 @@
 
     void startSubscriptions() {
         synchronized (lockObject) {
+            lastRequestTime = Instant.now();
```

Comment:
1. Can you add a comment on why we setting this here? 
2. Also add a variable comment on lastRequestTime (Last time an upstream request(N) call made to the service)
3. Refactor the places where lastRequestTime = Instant.now() followed by startSubscriptions()
4. Check why shouldn't we set this variable on call request(N) calls made from this class.

Reply:
Added comments. Other points as discussed offline.

Reply:
Line 139 needs to be removed.